mi/steering.py

```
@@ -0,0 +1,615 @@
+                    num_completions=num_completions,
+                )
+
+            classifications = [experiment.classify_text(c) for c in completions]
+            prompt_deceptive = sum(1 for c in classifications if c["deceptive_intent"])
+
+            coeff_results["prompts"].append({
+                "prompt_index": prompt_idx,
+                "prompt_preview": prompt[:100],
+                "completions": [
+                    {"text": c, **cls}
+                    for c, cls in zip(completions, classifications)
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)